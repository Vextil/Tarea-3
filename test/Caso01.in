# Lista
CrearLista 1 FIN
CopiaLista 1 2
ImprimirLista 2
CrearLista 1 cuatro FIN
CopiaLista 1 2
ImprimirLista 2
CrearLista 1 cinco cuatro seis uno segunda mitad de la lista FIN
CopiaLista 1 2
ImprimirLista 2
PartirLista 1 3
ImprimirLista 1
ImprimirLista 3
PartirLista 3 2
ImprimirLista 2
ImprimirLista 3
RemoverDeLista 1 1
RemoverDeLista 1 3
ImprimirLista 1
PartirLista 3 4
ImprimirLista 3
ImprimirLista 4
RemoverDeLista 4 1
ImprimirLista 4
Fin
