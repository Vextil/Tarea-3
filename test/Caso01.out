1># Lista
2>OK
3>OK
4>Ordenada. Lista vacia. 
5>OK
6>OK
7>Ordenada. cuatro 
8>OK
9>OK
10>NO ordenada. cinco cuatro seis uno segunda mitad de la lista 
11>OK
12>Ordenada. cinco cuatro seis uno 
13>NO ordenada. segunda mitad de la lista 
14>OK
15>Ordenada. de la lista 
16>NO ordenada. segunda mitad 
17>cinco
18>uno
19>Ordenada. cuatro seis 
20>OK
21>Ordenada. segunda 
22>Ordenada. mitad 
23>mitad
24>Ordenada. Lista vacia. 
25>Fin
